### PR TITLE
Feature/statistics

### PR DESCRIPTION
**Pull request by bgawkuc**
Created user and admin statistics

### CODE REVIEW

File: src/Statistics/views.py [hunk 1]
```diff
@@ -45,4 +53,342 @@ def durationPlay(request):
...
+
+def addContentToMyReport(pdf, data, tableContentData, title, tableTitlesData, xLabel, yLabel, id_):
+    data1, data2, data3, pdf = addToPDF(pdf, data, title, tableTitlesData, tableContentData)
+    x = [str(i) for i in range(1, len(data1) + 1)]
+
+    if len(tableTitlesData) == 3:
+        createBarPlot(x, data2, title, xLabel, yLabel, id_)
+    elif len(tableTitlesData) == 7:
+        createBarPlot(x, data3, title, xLabel, yLabel, id_)
+
```
Comment (kappa243) [inline src/Statistics/views.py]: Incorrect length of _tableTitlesData_ causes image not to be created; unhandled exception.

File: src/Statistics/views.py [hunk 2]
```diff
@@ -45,4 +53,342 @@ def durationPlay(request):
...
+    pdf = createPDF()
+
+    tableTitlesData = ['Id', 7, 'Title', 50, 'Artist', 25, 'Count']
+    tableContentData = [10, 60, 26, 10]
+    addContentToMyReport(pdf, fiveBestSongs, tableContentData, 'Number of plays of my 5 best songs', tableTitlesData, 'Song ID', 'Number', 5)
+
+    tableTitlesData = ['Id', 7, 'Artist', 75, 'Count']
+    tableContentData = [10, 92, 0]
+    addContentToMyReport(pdf, fiveBestArtists, tableContentData, 'Number of plays of my best 5 artist\'s song', tableTitlesData, 'Artist', 'Number', 6)
+
```
Comment (kappa243) [inline src/Statistics/views.py:193]: _tableTitlesData_ has a length of 5 then is passed to **addContentToMyReport()**

**Review by kappa243 — CHANGES_REQUESTED** [review]
Unhandled exception to be fixed.

File: src/Statistics/views.py (hunk 2 above)
Comment (bgawkuc) [reply src/Statistics/views.py:193]: fixed this issue

**Review by kappa243 — APPROVED** [review]

**Pull request merged**